attestation-service/verifier/src/se/ibmse.rs

```diff
@@ -21,11 +21,13 @@ use serde::{Deserialize, Serialize};
 use serde_with::{base64::Base64, hex::Hex, serde_as};
 use std::{env, fs};
 
+const DEFAULT_CERTS_OFFLINE_VERIFICATION: &str = "false";
+
 const DEFAULT_SE_HOST_KEY_DOCUMENTS_ROOT: &str = "/run/confidential-containers/ibmse/hkds";
 
 const DEFAULT_SE_CERTIFICATES_ROOT: &str = "/run/confidential-containers/ibmse/certs";
 
-const DEFAULT_SE_CERTIFICATE_ROOT_CA: &str = "/run/confidential-containers/ibmse/DigiCertCA.crt";
+const DEFAULT_SE_CERTIFICATE_ROOT_CA: &str = "/run/confidential-containers/ibmse/root_ca.crt";
```

Comment:
The docs still refer to DigiCertCA.crt?

Reply:
@fitzthum thanks for reminding it, `DigiCertCA.crt` is signature referred in updated document here now: https://github.com/confidential-containers/trustee/pull/423/files#diff-cc40db36dd81eebf63e9ac05e577106665cc807a31579c53d77f25fd4a26642dR20